<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(208, 253, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60, 182, 182);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;ОРГАНИКА&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>201</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QPushButton" name="btn_alkanes">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Алканы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_alkenes">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Алкены</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_alkynes">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Алкины</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cycloalkanes">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Циклоалканы</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_alkadienes">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Алкадиены</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>201</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_ketones">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Альдегиды, кетоны</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_alcohols">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Спирты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_carboxylic">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Карбоновые кислоты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_amines">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Арены</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_phenol">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(178, 255, 234);</string>
       </property>
       <property name="text">
        <string>Фенол</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
